.buildkite/scripts/steps/integration_tests_tf.sh

```diff
@@ -11,6 +11,7 @@ asdf install
 
 GROUP_NAME=$1
 TEST_SUDO=$2
+TEST_FIPS=$3
```

Comment:
Do you think the value should be defaulted to `false`?

```suggestion
TEST_FIPS=${3:-false}
```

Reply:
I had it defaulting to `false` in the beginning but then changed it so it would have to be explicitly specified.  I figured a) that was clearer from the caller's end b) it was more future-proof if we added more positional arguments and c) it was consistent with the other arguments.